docs/md_book/book/ayu-highlight.css

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
we do not want this in the repo, this should be generated from CI and pushed to gh-pages